bootstrapers/hychart/utils/useTokenChecker.ts

```diff
@@ -12,19 +22,44 @@ export function useTokenChecker() {
     if (token) {
       auth.setToken(token as string);
       // Remove the 'token' query parameter from the URL
-      const { pathname, query, ...rest } = router;
+      const { pathname, query, ...rest } = router.query;
```

Comment:
Are you sure this is correct? do we have the query inside the query?

Reply:
yes, we have query inside router and this query has token